kudulite/Dockerfile

```diff
@@ -65,12 +65,14 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 # Use Dynamic Install SDK feature from Oryx build
 # Trim off all unnecessary SDKs and let them lazily loaded
 RUN rm -rf /opt/python /opt/hugo /opt/php /opt/php-composer \
-    # We still need the fundamental ones when building the images nodejs 9, npm 6 and dotnet 2.2
-    # TODO: This will be upgraded to dotnet 3.1 soon
```

Comment:
Would this be upgraded to dotnet 3.1 anytime soon?

Reply:
No, I follow Kiran's guide to retain the dotnet 3.1 SDKs in the oryx build image. It is not recommended to lazy load the dotnet SDK dynamically. Thus, the image itself now has all builtin dotnet SDKs.